helm/values-dev.yaml

```
@@ -150,6 +154,8 @@ microservice-chart:
     OTEL_EXPORTER_OTLP_HEADERS: elastic-otel-token-header
     NPG_API_KEY: npg-api-key
     NPG_CARDS_PSP_KEYS: npg-cards-psp-keys
+    NODE_FORWARDER_API_KEY: node-forwarder-api-key
+    REDIRECT_URL_MAPPING: redirect-refund-mapping
```

The mapping is generic, I would drop the `refund`

```suggestion
    REDIRECT_URL_MAPPING: redirect-url-mapping
```

---

I was thinking about that. The fact here is that the endpoint is taken from the configuration as is and then used to populate NodeForwarder api calls. This implementation does not use auto-generated openapi clients since communication is performed using NodeForwarder as a proxy.
For this reason i split the endpoints configurations as two different secrets, one for refund and one for redirect retrieve url ones. This can be collapsed to one single configuration by adding `/refunds` programmatically to retrieve urls endpoint. this, by the way, will limit configurations in the case refund urls differs from retrieve url ones (but this will be a specification drift).
IDK if leave it as a separate configuration or not. What do you think?

---

I think that leave a single configuration map is the best option. Fixed with https://github.com/pagopa/pagopa-ecommerce-event-dispatcher-service/pull/237/commits/3794d4c4084e9717d449ccc363dbab1d1c07c511